Package Managerの依存関係の不足

## 環境情報

- UniVRM version: `0.126.0`
- Unity version: `Unity-2022.3`
- OS: `Windows 11`

## バグについて

- バグの内容
コード「com.vrmc.gltf/Editor/TestRunner/TestRunner.cs」はPackage「Test Framework (com.unity.test-framework)」を必要としているが、依存関係が登録されていない
- ConsoleLog
```
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(10,32): error CS0234: The type or namespace name 'TestTools' does not exist in the namespace 'UnityEditor' (are you missing an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(6,7): error CS0246: The type or namespace name 'NUnit' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(8,19): error CS0234: The type or namespace name 'TestTools' does not exist in the namespace 'UnityEditor' (are you missing an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(26,38): error CS0246: The type or namespace name 'ICallbacks' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(52,36): error CS0246: The type or namespace name 'ITestAdaptor' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(60,37): error CS0246: The type or namespace name 'ITestResultAdaptor' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(85,37): error CS0246: The type or namespace name 'ITestAdaptor' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(89,38): error CS0246: The type or namespace name 'ITestResultAdaptor' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(103,53): error CS0246: The type or namespace name 'ITestResultAdaptor' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.vrmc.gltf@0.126.0\Editor\TestRunner\TestRunner.cs(103,28): error CS0246: The type or namespace name 'TNode' could not be found (are you missing a using directive or an assembly reference?)
```
- スクリーンショット
![image](https://github.com/user-attachments/assets/bc66d5da-daf3-4bf6-bdfa-7c4e67d7dce4)

- 再現方法
Unity Package「Test Framework 」が導入されていないUnityプロジェクトで、任意の方法でUniVRMの導入を行う
- エラーの発生するファイル(ここに貼り付けても差し支えの無いもの)
https://github.com/vrm-c/UniVRM/blob/master/Assets/UniGLTF/Editor/TestRunner/TestRunner.cs

--- a/Assets/UniGLTF/package.json
+++ b/Assets/UniGLTF/package.json
@@ -11,7 +11,8 @@
     "name": "VRM Consortium"
   },
   "dependencies": {
-    "com.unity.modules.animation": "1.0.0"
+    "com.unity.modules.animation": "1.0.0",
+    "com.unity.test-framework": "1.0.0"
   },
   "samples": [
     {

